### ISSUE TITLE
Add to Cart : Error : Page Reloads

### ISSUE THREAD

**Issue by kavania2002** [kavania2002/MobiMart#33]
**Describe the bug**
When we click on add to cart button, the page reloads and then the theme gets changed to default i.e. white theme

**Expected behavior**
After clicking the add to cart button, the theme should remain the same.

**Comment by Sloth-Panda** [comment]
would like to work on this @kavania2002

**Comment by kavania2002** [comment]
Yeah sure, you may work on this issue first. After that I will assign others.

**Comment by Sloth-Panda** [comment]
@kavania2002  these are the set of commands to get started with the project working on our system right?
1. pip install requirements.txt
2. cd Mobimart
3. python  manage.py migrate
4.  python manage.py runserver

**Comment by kavania2002** [comment]
yes, u r right?
if u don't want use a virtual environment then this is correct.
and u can join discord channel (link in readme) for any other issues. I will guide you properly.

**Comment by kavania2002** [comment]
hey @Sloth-Panda, what's the status?

**Comment by kavania2002** [comment]
@arbazalam01 would like to do this?
